### PR TITLE
use ftw.referencewidget

### PR DESCRIPTION
**Pull request by tschanzt**

### CODE REVIEW

File: ftw/topics/behavior.py [hunk 1]
```diff
@@ -3,46 +3,34 @@
...
-from plone.formwidget.contenttree import MultiContentTreeFieldWidget
-from plone.formwidget.contenttree import ObjPathSourceBinder
-from plone.formwidget.contenttree.utils import closest_content
 from z3c.relationfield.event import _setRelation
-from z3c.relationfield.schema import RelationChoice
 from z3c.relationfield.schema import RelationList
 from zope.component import getMultiAdapter
 from zope.interface import alsoProvides
 from zope.interface import Interface
+from ftw.referencewidget.widget import ReferenceBrowserWidget
```
Comment (mbaechtold) [inline ftw/topics/behavior.py]: The list of imports seems to be sorted alphabetically. I tend to adhere to the local consistency in this case.

File: ftw/topics/upgrades/20160830111534_install_refwidget/upgrade.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from ftw.upgrade import UpgradeStep
+
+
+class InstallRefwidget(UpgradeStep):
+    """Install refwidget.
```
Comment (mbaechtold) [inline ftw/topics/upgrades/20160830111534_install_refwidget/upgrade.py]: Suggestion for the doctoring: `Install reference browser widget from "ftw.referencewidget"`

**Comment by maethu** [comment]
Please wait with a merge until first release of ftw.referencewidget.

File: sources.cfg [hunk 3]
```diff
@@ -4,3 +4,7 @@ extensions = mr.developer
 
 auto-checkout =
     ftw.testing
+    ftw.referencewidget
```
Comment (mbaechtold) [inline sources.cfg]: 🚧

File: sources.cfg [hunk 4]
```diff
@@ -4,3 +4,7 @@ extensions = mr.developer
...
+
+[branches]
+ftw.referencewidget = tt-impl
```
Comment (mbaechtold) [inline sources.cfg]: 🚧

**Comment by mbaechtold** [comment]
@Tschanzt, could please remove the installation from source and add an entry to the changelog?

**Comment by Tschanzt** [comment]
@mbaechtold updated.

**Comment by mbaechtold** [comment]
👍

**Pull request merged**